### PR TITLE
[SHELL32] Move to/Copy to folder should not be displayed when unable to retrieve the file name

### PR DESCRIPTION
**Pull request by TAN-Gaming**
## Purpose
JIRA issue: [CORE-18909](https://jira.reactos.org/browse/CORE-18909)

### CODE REVIEW

**Review by KRosUser2 — APPROVED** [review]
Lgtm, better than my patch

File: dll/win32/shell32/CCopyToMenu.cpp [hunk 1]
```diff
@@ -192,7 +192,7 @@ HRESULT CCopyToMenu::DoRealCopy(LPCMINVOKECOMMANDINFO lpici, LPCITEMIDLIST pidl)
 
 CStringW CCopyToMenu::DoGetFileTitle()
 {
-    CStringW ret = L"(file)";
+    CStringW ret; // Empty
```
Comment (learn-more) [inline dll/win32/shell32/CCopyToMenu.cpp:195]: What is the point of the comment?

Comment (TAN-Gaming) [reply dll/win32/shell32/CCopyToMenu.cpp:195]: It's easier to understand the code: returns empty string in case of failure.

Comment (TAN-Gaming) [reply dll/win32/shell32/CCopyToMenu.cpp]: Removed.

**Review by JoachimHenze — APPROVED** [review]
LGTM now. Retval hr is preintialized to E_FAIL in both cases.

File: dll/win32/shell32/CCopyToMenu.cpp [hunk 2]
```diff
@@ -240,6 +240,9 @@ HRESULT CCopyToMenu::DoCopyToFolder(LPCMINVOKECOMMANDINFO lpici)
     }
 
     CStringW strFileTitle = DoGetFileTitle();
+    if (strFileTitle.IsEmpty())
```
Comment (learn-more) [inline dll/win32/shell32/CCopyToMenu.cpp:243]: I am not convinced that the _title_ of a dialog should dictate whether or not we can copy or move a file.

Comment (KRosUser2) [reply dll/win32/shell32/CCopyToMenu.cpp:243]: @learn-more , we refer to file title (name without extension) not dialog title... Too fast in your comments guy !

Comment (KRosUser2) [reply dll/win32/shell32/CCopyToMenu.cpp:243]: Ok so everything is fine.

Comment (learn-more) [reply dll/win32/shell32/CCopyToMenu.cpp:243]: You didn't convince me with your weird arguments, so no, I am still not convinced that the title used for a dialog should be used to check if we can copy a file.

**Review by learn-more — CHANGES_REQUESTED** [review]
I am not convinced that something that is used as title of a dialog should dictate whether or not we can copy or move a file.

I suspect that it should be more something like is being done in `CDefaultContextMenu::QueryContextMenu`, where actual attributes (https://learn.microsoft.com/en-us/windows/win32/shell/sfgao) are being checked.

**Comment by KRosUser2** [comment]
@TAN-Gaming : Reference to my patch to be removed from the PR description as it's no longer being used nor the best option.

**Comment by TAN-Gaming** [comment]
Maybe next time.

**Pull request closed without merging**